app/javascript/menu/group-switcher.jsx

```diff
@@ -16,9 +17,9 @@ export const GroupSwitcher = ({ miqGroups, currentGroup, expanded: isExpanded })
     value: currentGroup.id,
   };
 
-  const groupChange = ({ selectedItem: { value: group_id } }) => {
```

Comment:
Will this change break something, looking at https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/menu/group-switcher.jsx#L14-L17 that is setting a `value` key

Reply:
No, this is a harmless rename...